code/game/objects/items/pinpointer.dm

```diff
@@ -152,6 +152,74 @@
 	if(!target) //target can be set to null from above code, or elsewhere
 		active = FALSE
 
+/obj/item/pinpointer/tracker //A modified pinpointer that tracks mobs with tracking implants
+	name = "tracking implant pinpointer"
+	desc = "A handheld tracking device that points to individuals with active tracking implants."
+	icon_state = "pinpointer_tracker"
+	var/has_owner = FALSE
+	var/pinpointer_owner = null
+
+/obj/item/pinpointer/tracker/proc/implanted(mob/living/H)
+	var/turf/here = get_turf(src)
+	for(var/obj/item/implant/I in H.implants)
+		if(istype(I, /obj/item/implant/tracking))
+			continue
```

Comment:
```suggestion
		if(!istype(I, /obj/item/implant/tracking))
			continue
```
continue is like a return but for a loop basically, from what i'm seeing you don't want this here